src/ViewLayer/DebugDisplay/FaultPage/FaultView/FaultView.cpp

```diff
@@ -81,10 +81,10 @@ FaultView::~FaultView()
 {
 }
 
-void FaultView::initializeLabel(QLabel& label, QLayout*& layout, QString& styleSheet)
+void FaultView::initializeLabel(FaultLabel& label, QLayout*& layout, QString& styleSheet)
 {
     label.resize(WIDTH, HEIGHT);
-    label.setStyleSheet(styleSheet);
+    label.setStyleSheet(styleSheet + label.color().name());
```

Comment:
Let's use the `.arg` function for this instead.
http://doc.qt.io/qt-5/qstring.html#arg